release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -3069,6 +3069,15 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.17.11 --pullspecs
 ----
 
+[id="ocp-4-17-11-enhancements_{context}"]
+==== Enhancements
+
+[id="ocp-4-17-11-wif-storage-support_{context}"]
+===== GCP Filestore supporting Workload Identity is generally available
+Google Compute Platform (GCP) Filestore Container Storage Interface (CSI) storage supports Workload Identity. This allows users to access Google Cloud resources using federated identities instead of a service account key. This feature is generally available.
```

Comment:
```suggestion
Google Compute Platform (GCP) Filestore Container Storage Interface (CSI) storage supports Workload Identity. You can now access Google Cloud resources by using federated identities instead of a service account key. This feature is generally available.
```

Avoid using "allow": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a

> Avoid stating that inanimate objects grant abilities to people, as in “the product allows you to…”. Whenever possible, use a direct, user-focused alternative such as “you can use the product to…” or “with this product, users can…”.